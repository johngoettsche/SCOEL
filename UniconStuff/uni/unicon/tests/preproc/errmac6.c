/* test error message: stringizing operator at end of macro */

#define f(a) a#
f(1)
