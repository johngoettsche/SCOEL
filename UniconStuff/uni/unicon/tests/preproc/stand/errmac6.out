pp: File errmac6.c; Line 3: # preprocessing expression must not cross directive boundary
